gateway/1.8/modules/ROOT/pages/flex-gateway-istio.adoc

```diff
@@ -0,0 +1,540 @@
+= Integrating Flex Gateway with Istio Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Integrate Flex Gateway into your Istio Service Mesh to leverage Flex Gateway policies, logging, and metrics from the Anypoint Platform control plane. Deploy Flex Gateway as an ingress gateway to manage incoming north-south cluster traffic and internal east-west traffic inside the cluster.
+
+To deploy this Flex Gateway use case, use the following architecture diagram and tutorial. The use case is a modified version of the https://istio.io/latest/docs/examples/bookinfo/[Istio Bookinfo Application] with Flex Gateway integrated. In the use case:
+
+* An Istio virtual service forwards all ingress north-south traffic to Flex Gateway.
+* Direct access to the `bookinfo` name space is restricted. All external traffic into the `bookinfo` namespace must come from the `flex` namespace. Istio automatically configures mTLS configuration between the `bookinfo` and `flex`.
+* Flex Gateway protects and monitors the exposed product page service from incoming traffic with an API instance.
+* The Istio sidecar intercepts east-west traffic between the product page service and the reviews service. Flex Gateway acts as an ingress gateway managing incoming traffic to the reviews service.
+
+image:flex-istio-deployment.png["A detailed image of the previously described use case."]
```

Comment:
The alt text should be a sentence fragment that describes the image, but not be self-referential. (Don't say "A detailed image")

Can you try writer.ai to generate alt text?